beesdoo_easy_my_coop/views/subscription_templates.xml

```diff
@@ -4,28 +4,29 @@
 
 <odoo>
 
-   <template id="beesdoo_easy_my_coop.becomecooperator"
-             name="Become Cooperator (in beesdoo_easy_my_coop)"
-             inherit_id="easy_my_coop_website.becomecooperator">
-		<xpath expr="//div[@id='internal_rules_approved']" position="before">
-			<div id="info_session_confirmed" t-if="display_info_session" t-attf-class="form-group" >
-				<label class="col-md-3 col-sm-4 control-label" 
-				       for="info_session_confirmed">
-				       		Info Session
-				</label>
-				<div class="col-md-9 col-sm-8">
-					<div class="checkbox">
-						<label for="info_session_confirmed">
-							<input type="checkbox"
-							       name="info_session_confirmed"
-								   t-att-required="info_session_required"
-								   t-att-checked="data_policy_approved"/>
-							<t t-raw="info_session_text"/>
-						</label>
-					</div>
-				</div>
-			</div>
-		</xpath>
-  </template>
+    <template id="beesdoo_easy_my_coop.becomecooperator"
+              name="Become Cooperator (in beesdoo_easy_my_coop)"
+              inherit_id="easy_my_coop_website.becomecooperator">
+        <xpath expr="//div[@id='internal_rules_approved']" position="before">
+            <div id="info_session_confirmed" t-if="display_info_session" t-attf-class="form-group" >
+                <label class="col-md-3 col-sm-4 control-label"
+                    for="info_session_confirmed">
+                    Info Session
+                </label>
+                <div class="col-md-9 col-sm-8">
+                    <div class="form-check">
+                        <input type="checkbox"
+                               name="info_session_confirmed"
+                               class="form-check-input"
+                               t-att-required="info_session_required"
+                               t-att-checked="data_policy_approved"/>
```

Comment:
Looking at https://github.com/coopiteasy/vertical-cooperative/pull/129 I think this `data_policy_approved ` should maybe be `info_session_confirmer`? Although it was like this before this PR...

Reply:
Totaly right ! I'll fix that ! :) Thanks for your review.